### PR TITLE
`kpt fn render` includes local and meta resources by default

### PR DESCRIPTION
**Pull request by droot**
This PR adds support for including `local` and `meta` resources by default in `fn render`.

To exclude meta resource (just the `Kptfile`), users can run `kpt fn render --exclude-meta-resources` .

### CODE REVIEW

File: internal/cmdrender/executor.go [hunk 1]
```diff
@@ -92,12 +92,18 @@ func (e *Executor) Execute(ctx context.Context) error {
 	at := attribution.Attributor{Resources: hctx.root.resources, CmdGroup: "fn"}
 	at.Process()
 
+	includeMetaResources := true
```
Comment (yuwenma) [inline internal/cmdrender/executor.go:95]: will this eventually be changed as a configurable flag? Or kpt always include meta resources in `fn render`

**Comment by droot** [comment]
ref: https://github.com/GoogleContainerTools/kpt/issues/2590

File: internal/cmdrender/executor.go (hunk 1 above)
Comment (droot) [reply internal/cmdrender/executor.go:95]: Actually, thought some more and realized exposing this as `configurable` flag will make it consistent with other `fn` commands and also provide a way to address the breaking change. Users can run `fn render --include-meta-resources false` to get the previous behavior. I will update the PR.

**Comment by droot** [comment]
Quick update: Now this PR is against the target feature branch `variant-constructor` where we plan to accumulate all changes related to `variant constructor pattern` /cc @yuwenma @bgrant0607

File: internal/cmdrender/executor.go [hunk 2]
```diff
@@ -168,6 +176,10 @@ type hydrationContext struct {
 	// imagePullPolicy controls the image pulling behavior.
 	imagePullPolicy fnruntime.ImagePullPolicy
 
+	// indicate if package meta resources such as Kptfile
+	// to be excluded from the function in put during render.
+	encludeMetaResources bool
```
Comment (yuwenma) [inline internal/cmdrender/executor.go:181]: ditto

**Pull request merged**